src/view/components/rule-engine/rule-engine.container.ts

```diff
@@ -91,6 +93,8 @@ export class RuleEngineContainer {
   rules$:EventEmitter<RuleModel[]> = new EventEmitter()
   ruleActions$:EventEmitter<ActionModel[]> = new EventEmitter()
   conditionGroups$:EventEmitter<ConditionGroupModel[]> = new EventEmitter()
+  globalError:string;
+  showRules:boolean
```

Comment:
Are you actually using this field? It looks like you're setting the value on the 'state' object, not 'this'.